state-sync/storage-service/server/src/tests.rs

```diff
@@ -124,9 +152,14 @@ async fn test_get_storage_server_summary() {
                 highest_version,
             )),
             epoch_ending_ledger_infos: Some(CompleteDataRange::from_genesis(highest_epoch - 1)),
-            transactions: Some(CompleteDataRange::from_genesis(highest_version)),
-            transaction_outputs: Some(CompleteDataRange::from_genesis(highest_version)),
-            account_states: Some(CompleteDataRange::from_genesis(highest_version)),
+            transactions: Some(CompleteDataRange::new(FIRST_TXN_VERSION, highest_version).unwrap()),
+            transaction_outputs: Some(
+                CompleteDataRange::new(FIRST_TXN_OUTPUT_VERSION, highest_version).unwrap(),
+            ),
+            account_states: Some(
+                CompleteDataRange::new(LAST_TXN_VERSION - STATE_PRUNE_WINDOW + 1, highest_version)
```

Comment:
Yeah, much better (51) was random without the context of that it's the window +1